### DISCUSSION TITLE
Add non supported properties in site columns

### DISCUSSION THREAD

**Discussion by cpercarb** [pnp/pnpcore#1584] [Q&A]
Hello. My team and I we are just wondering if you have in the roadmap or is it going to be included in future versions the posibility to add currently non supported properties for specific site columns. Just to point an example: in a number colum, these properties are not available through the field interface.

![image](https://github.com/user-attachments/assets/a3a6c1a9-109e-4761-9db4-fb2aa3a661d4)
![image](https://github.com/user-attachments/assets/e101b9c9-3b1c-4994-b817-a7fbd3cdd51a)

This is just an example. 
Thanks in advance.

**Comment by jansenbe** [comment]
You can always craft the field schema XML yourselves and use https://pnp.github.io/pnpcore/using-the-sdk/fields-add.html#add-field-via-field-xml. A bit more work, but hopefully can get you unblocked
